src/serializers/html.js

```diff
@@ -124,6 +124,14 @@ class Html {
       return memo
     }, [])
 
+    if (nodes.length === 0) {
+      nodes = [{
+        kind: 'block',
+        type: 'paragraph',
```

Comment:
Hey @vpontis can you change this to make `type` use the `defaultBlockType`, similar to how it's done a few lines above? Thanks!